ObjectOrientedPractics/OOP/Model/Category.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOP.Model
+{
+    /// <summary>
+    /// Перечисление, содержащее категории товаров.
+    /// </summary>
+    public enum Category
```

Comment:
Лучше уточнить, так как категория слишком общее слово, она может быть не только у товара.
```suggestion
    public enum ItemCategory
```